bond_test.go

```diff
@@ -51,10 +63,23 @@ type User struct {
 	Username  string `db:"username"`
 }
 
+func (u User) AfterCreate(ctx bond.Session) {
```

Comment:
I actually think `sess` is better, because it does not require any further explanation :).